pkg/controller/studyjob/utils.go

```diff
@@ -39,6 +39,58 @@ func createWorkerJobObj(kind string) runtime.Object {
 	return nil
 }
 
+func validateWorkerResource(wkind string) error {
+	for _, crd := range invalidCRDResources {
+		if crd == wkind {
+			return fmt.Errorf("Cannot support %s; If CRD for it installed, please restart studyjob-controller to take effect", wkind)
```

Comment:
Maybe specifically say "Please install the CRD and restart studyjob-controller"?

Reply:
Thanks! updated